### PR TITLE
Helm Charts guide

### PR DESCRIPTION
**Pull request by aj-jester**
Goes with https://github.com/replicatedhq/kotsapps/pull/24

### CODE REVIEW

File: content/vendor/guides/helm-chart.md [hunk 1]
```diff
@@ -28,83 +27,219 @@ A full example of the code for this guide can be found in the [kotsapps reposito
...
-##### With helm package
+Add a Helm repo
 
-- git checkout https://github.com/helm/charts
-- cd charts/stable/grafana
-- you can use the latest, but we'll be working off of this commit: `git checkout f2fcadaa5` (just to be sure)
-- helm package .
-- VALIDATE: tar xOf grafana-5.0.11.tgz grafana/Chart.yaml, you should see
+```shell
+helm repo add stable https://kubernetes-charts.storage.googleapis.com
```
Comment (dexhorthy) [inline content/vendor/guides/helm-chart.md:37]: when showing single-line shell commands with their output, I tend to try to put a `$` at the start of the line for the commands, just to visually differentiate between the command and the output

```suggestion
$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
```

File: content/vendor/guides/helm-chart.md [hunk 2]
```diff
@@ -28,83 +27,219 @@ A full example of the code for this guide can be found in the [kotsapps reposito
...
-  name: rtluckie
-- email: maor.friedman@redhat.com
-  name: maorfr
+"stable" has been added to your repositories
+```
+
+Verify the repo has been added
+
+```shell
+helm repo ls
```
Comment (dexhorthy) [inline content/vendor/guides/helm-chart.md:45]: ditto the `$`

Comment (dexhorthy) [reply content/vendor/guides/helm-chart.md:45]:
```suggestion
$ helm repo ls
```

File: content/vendor/guides/helm-chart.md [hunk 3]
```diff
@@ -28,83 +27,219 @@ A full example of the code for this guide can be found in the [kotsapps reposito
...
+Clone the upstream `helm/charts` repo.
+
+```shell
+git clone git@github.com:helm/charts.git
+```
+
+Helm package the Grafana install.
+
+```shell
+helm package charts/stable/grafana
```
Comment (dexhorthy) [inline content/vendor/guides/helm-chart.md:68]:
```suggestion
$ helm package charts/stable/grafana
```

File: content/vendor/guides/helm-chart.md [hunk 4]
```diff
@@ -28,83 +27,219 @@ A full example of the code for this guide can be found in the [kotsapps reposito
...
+
+Go to `kotsadm` and click on `Version history -> Check for updates`. Once the update is available click on `Deploy`.
+
+![Helm Chart Check for Updates 1](/images/guides/kots/helm-chart-check-for-updates-1.png)
+
+
+Verify Grafana chart is deployed
+
+```shell
+kubectl get po
```
Comment (dexhorthy) [inline content/vendor/guides/helm-chart.md:129]:
```suggestion
$ kubectl get po
```

**Comment by aj-jester** [comment]
Comments addressed, merging.

**Pull request merged**